lapis/src/main/kotlin/org/genspectrum/lapis/controller/LapisController.kt

```diff
@@ -1149,6 +1165,88 @@ class LapisController(
         )
     }
 
+    @GetMapping(PHYLO_SUBTREE_ROUTE, produces = [TEXT_NEWICK_VALUE])
+    @LapisPhyloSubtreeResponse
+    fun getPhyloSubtreeAsNewick(
+        @PrimitiveFieldFilters
+        @RequestParam
+        sequenceFilters: GetRequestSequenceFilters?,
+        @PhyloTreeField
+        @RequestParam
+        phyloTreeField: String,
+        @NucleotideMutations
+        @RequestParam
+        nucleotideMutations: List<NucleotideMutation>?,
+        @AminoAcidMutations
+        @RequestParam
+        aminoAcidMutations: List<AminoAcidMutation>?,
+        @TreeDataFormatParam
+        @RequestParam
+        dataFormat: String? = null,
+        @NucleotideInsertions
+        @RequestParam
+        nucleotideInsertions: List<NucleotideInsertion>?,
+        @AminoAcidInsertions
+        @RequestParam
+        aminoAcidInsertions: List<AminoAcidInsertion>?,
+        @RequestHeader httpHeaders: HttpHeaders,
+        response: HttpServletResponse,
+    ) {
+        val request = PhyloTreeSequenceFiltersRequest(
+            sequenceFilters?.filterKeys { !SPECIAL_REQUEST_PROPERTIES.contains(it) }
+                ?: emptyMap(),
+            nucleotideMutations ?: emptyList(),
+            aminoAcidMutations ?: emptyList(),
+            nucleotideInsertions ?: emptyList(),
+            aminoAcidInsertions ?: emptyList(),
+            phyloTreeField = validatePhyloTreeField(
+                phyloTreeField,
+                caseInsensitiveFieldConverter,
+                databaseConfig,
+            ).fieldName,
+        )
+
+        TreeDataFormat.fromAcceptHeaders(httpHeaders.accept)
```

Comment:
Should the return value be used?

Reply:
Ah its actually not used at the moment as the only tree format that is used is NEWICK - so the function can actually be removed - I will do that before merging